src-electron/db/query-config.js

```diff
@@ -235,7 +235,10 @@ async function insertOrUpdateAttributeState(
     )
     staticAttribute.defaultValue = featureMapDefaultValue
   }
-  let forcedExternal = await queryUpgrade.getForcedExternalStorage(db)
+  let forcedExternal = await queryUpgrade.getForcedExternalStorage(
+    db,
+    staticAttribute.packageRef
```

Comment:
[staticAttribute.packageRef]

Reply:
thanks I fixed this in getForcedExternalStorage